src/harness/client.ts

```diff
@@ -95,7 +95,7 @@ interface RenameEntry {
 }
 
 /** @internal */
-export function extractMessage(message: string): string {
+function extractMessage(message: string): string {

```

Comment:
I think I said this before and I just haven't followed up on it, but I definitely want to edit our jsdoc eslint rule to ban `@internal` on unexported declarations. Not for you to do, but something I think should go in first so that we know to drop these comments.